### DISCUSSION TITLE
Installation via Snap and Flathub

### DISCUSSION THREAD

**Discussion by tracker1** [zoidsh/meru#253] [Feature Requests] [answered]
Would be nice to see this in Flathub and/or Snap for easy Linux install/discovery.

**Comment by cig0** [comment]
HI,

I downloaded the Snap package **gmail-desktop-2.20.4-linux.snap** to give it try, got the following error message:

`error: cannot find signatures with metadata for snap "gmail-desktop-2.20.4-linux.snap"`

Could it be isn't published on the Snapcraft store? At least, I couldn't find it there.

**Comment by timche** [comment]
We haven't submitted it to the Snapcraft store. I'm not using Linux, so we'd need some support here to manage different package managers.

**Comment by timche** [comment]
I'll try to release it in the Snap Store, but I'll need to get ownership of the [currently registered snap](https://snapcraft.io/gmail-desktop). Will get back to you later.

**Comment by timche** [comment, marked as answer]
Okay, apparently the [Gmail Desktop in the Snap Store](https://snapcraft.io/gmail-desktop) is actually not this Gmail Desktop, but another one and there's no chance we release there under the same name. That's unfortunate, but luckily with v3 it's planned that Gmail Desktop will be renamed anyway (#242), so Gmail Desktop will be released on Snap soon under another name.
